pkgs/development/python-modules/fmpy/default.nix

```diff
@@ -0,0 +1,153 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, cmake
+, attrs
+, lark-parser
+, lxml
+, msgpack
+, numpy
+, scipy
+, pytz
+, dask
+, requests
+, matplotlib
+, pyqt5
+, pyqtgraph
+, notebook
+, plotly
+}:
+let
+  # Auxiliary library, maybe not extremely useful by itself.
```

Comment:
I think we should move them anyway to all-packages just incase they are used somewhere else down the line.

Reply:
Done. I had to add more options to `sundials` package to make it more compliant with the FMPy. Hope that's fine.